content/en/news/support/announcing-1.3-eol-final/index.md

```diff
@@ -0,0 +1,12 @@
+---
+title: Support for Istio 1.3 has ended
+subtitle: Support Announcement
+description: Istio 1.3 end of life announcement.
+publishdate: 2020-02-14
+aliases:
+    - /news/2020/announcing-1.3-eol-final
```

Comment:
Do we need an alias? I admit I'm not sure how to verify that we ever used the prior link, but mentioning this in case.

Reply:
I'm using the same structure and alias pattern as for the 1.2 final EOL announcement. This should work.

Reply:
Well, I think that is because back then we had a content/en/news/2019 folder that had things in it, and the year folders were removed which is why we had to add the alias (so both links would work). (I looked at the release-1.3 branch to see the old directory structure). I don't think this doc ever existed at the news/2020 location as it was post that structure change, so we shouldn't need to alias.

Reply:
The original 1.2 EOL announce was at the news/2019 location: `https://github.com/istio/istio.io/pull/5599/files`.  In https://github.com/istio/istio.io/pull/5740 it was moved from news/2019 to news/support and the alias added at that time. 

For 1.3, it never existed at a news/2020 and moved, so we shouldn't need the alias.

Reply:
Sounds good. I removed it.